ExtendElementInSubAccounts/util.js

```diff
@@ -1,6 +1,12 @@
 const axios = require('axios');
```

Comment:
On line 22, case 'PROD' need to be replaced with case 'US_PROD', for script to work in production.
Also, i think on line 24, case 'PROD_EU' need to be replaced with case 'EU_PROD'.

Reply:
Good catch, thanks!